CHANGELOG.md

```diff
@@ -5,10 +5,78 @@ we recommend most users use the latest `master` branch of Teku.
 
 ## Upcoming Breaking Changes
 
-- REST API endpoints will be updated to match emerging standards in a future release.
+- REST API endpoints will be updated to match emerging standards in a future release. Current replacements:
+  - GET `/eth/v1/beacon/states/{state_id}/validators/{validator_id}` replaced by POST `/eth/v1/beacon/states/{state_id}/validators/{validator_id}`
+  - `/network/enr` replaced by `/eth/v1/node/identity`
+  - `/network/listen_addresses` replaced by `/eth/v1/node/identity`
+  - `/network/peer_id` replaced by `/eth/v1/node/identity`
+  - `/network/peers` replaced by `/eth/v1/node/peers`
+  - `/network/peer_count` replaced by `/eth/v1/node/peers`
+  - `/network/listen_port` replaced by `/eth/v1/node/identity`
+  - `/node/fork` replaced by `/eth/v1/beacon/states/{state_id}/fork`
+  - `/node/genesis_time` replaced by `/eth/v1/beacon/genesis`
+  - `/node/syncing` replaced by `/eth/v1/node/syncing`
+  - `/node/version` replaced by `/eth/v1/node/version`
+  - GET `/validator/attestation` replaced by `/eth/v1/validator/attestation_data`
+  - POST `/validator/attestation` replaced by `/eth/v1/beacon/pool/attestations`
+  - GET `/validator/block` replaced by `/eth/v1/validator/blocks/{slot}`
+  - POST `/validator/block` replaced by `/eth/v1/beacon/blocks`
+  - `/validator/aggregate_attestation` replaced by `/eth/v1/validator/aggregate_attestation`
+  - POST `/validator/duties` replaced by POST `/eth/v1/validator/duties/attester/{epoch}` and GET `/eth/v1/validator/duties/proposer/{epoch}`
+  - `/validator/aggregate_and_proofs` replaced by `/eth/v1/validator/aggregate_and_proofs`
+  - `/validator/beacon_committee_subscription` replaced by `/eth/v1/validator/beacon_committee_subscriptions`
+  - `/validator/persistent_subnets_subscription` deprecated. The beacon node now automatically establishes persistent subnet subscriptions based on calls to `/eth/v1/validator/beacon_committee_subscriptions`
 - `--validators-key-files` and `--validators-key-password-files` have been replaced by `--validator-keys`. The old arguments still work but will be removed in a future release.
 - Validator subcommands for generating and registering validators are now deprecated and will be removed in a future release to encourage the use of the Eth2 Launchpad, which is the most secure way of generating keys and sending deposits.
 
+## 0.12.11
+
+### Additions and Improvements
+- Teku now supports running the validator client as a separate process via the `teku validator-client` subcommand. The in-process validator client is still supported. 
+  
+  **IMPORTANT**: Ensure validator keys are only passed as arguments to *either* the beacon node or the validator client but never both at the same time.
```

Comment:
is there something i should look out for to indicate that i've passed the same key to both, and what is the likely outcome if i get it wrong?

Reply:
I think that level of detail is more for the docs.  The main thing is to make people aware that they shouldn't do it. Made it clear that this is to avoid a risk of slashing though.